### PR TITLE
Feature/linter

### PR DESCRIPTION
**Pull request by alexfradiani**
Adding standardized style formatting with eslint + prettier:

- Commonly used guidelines, e.g., 80-column lines, 2-spaces indentation, semicolons, single-quotes, etc.
- All files have the same code..., they have just been refreshed with the formatting.

### CODE REVIEW

**Review by Alxzu — APPROVED** [review]
Awesome

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "proseWrap": "preserve",
+  "quoteProps": "as-needed",
+  "requirePragma": false,
+  "semi": true,
+  "singleQuote": true,
+  "tabWidth": 2,
+  "trailingComma": "none",
+  "useTabs": false,
+  "vueIndentScriptAndStyle": false,
+  "filepath": "/Users/alexfradiani/dev/2020/RS/express-api-base/__tests__/controllers/user.controller.spec.ts",
```
Comment (alealvarez14) [inline .prettierrc]: maybe use only path from source?

File: src/database/factories/user.factory.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
-import * as Faker from "faker";
-import { define } from "typeorm-seeding";
-import { User } from "../../entities/user.entity";
+import * as Faker from 'faker';
+import { define } from 'typeorm-seeding';
+import { User } from '../../entities/user.entity';
```
Comment (alealvarez14) [inline src/database/factories/user.factory.ts]: We can use `@entities/user.entity` here

File: src/database/seeds/users.seed.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
-import { User } from '../../entities/user.entity'
-import { Seeder, Factory } from 'typeorm-seeding'
+import { User } from '../../entities/user.entity';
```
Comment (alealvarez14) [inline src/database/seeds/users.seed.ts]: Same as above

File: .prettierrc (hunk 1 above)
Comment (alexfradiani) [reply .prettierrc]: ooops... will fix it

**Review by alealvarez14 — APPROVED** [review]

File: src/app.ts [hunk 4]
```diff
@@ -1,50 +1,55 @@
...
-import { Container } from "typedi";
-import { createConnection } from "typeorm";
+import 'reflect-metadata'; // this shim is required
+import { createExpressServer, useContainer } from 'routing-controllers';
+import { Container } from 'typedi';
+import { createConnection } from 'typeorm';
 import AdminBro from 'admin-bro';
 //TODO: Check how to change this to TS import
-const AdminBroExpress = require('@admin-bro/express')
+const AdminBroExpress = require('@admin-bro/express');
```
Comment (nsantos16) [inline src/app.ts:7]: I think for handle this type of imports you will need to configure `tsconfig.json` and  modify the `baseUrl` attribute to `.` as well as the other paths according to this path, in this way you can include `admin-bro`

**Review by nsantos16 — APPROVED** [review]

File: src/app.ts (hunk 4 above)
Comment (alexfradiani) [reply src/app.ts:7]: Thanks Nico, will take a look.

**Review by diebarral — APPROVED** [review]

**Pull request merged**